simulator/fault_injection/CMakeLists.txt

```diff
@@ -42,16 +42,23 @@ if(BUILD_TESTING)
     fault_injection_node_component
   )
 
-  # TODO: Comment out until the error in CI is resolved.
   # launch_testing
-  # find_package(launch_testing_ament_cmake REQUIRED)
-  # add_launch_test(test/test_fault_injection_node.test.py)
-
-  # install(DIRECTORY
-  #   test/config
-  #   test/launch
-  #   DESTINATION share/${PROJECT_NAME}/
-  # )
+  find_package(launch_testing_ament_cmake REQUIRED)
+  add_launch_test(
+    test/test_fault_injection_node.test.py
+    TARGET test_fault_injection_launch_test
+  )
+  set_tests_properties(
```

Comment:
How about adding a comment here such as `TODO: Remove in Humble`?

Reply:
Good. Added a comment : 80480e333